boards/arm/efm32pg_stk3401a/doc/index.rst

```diff
@@ -0,0 +1,189 @@
+
+EFM32 Pearl Gecko Starter Kit
+#############################
+
+Overview
+********
+
+The EFM32 Pearl Gecko Starter Kit EFM32PG-STK3401A contains an MCU from the
+EFM32PG family built on an ARM® Cortex®-M4F processor with excellent low
+power capabilities.
+
+.. figure:: ./efm32pg_stk3401a.jpg
+   :width: 375px
+   :align: center
+   :alt: EFM32PG-SLSTK3401A
+
+   EFM32PG-SLSTK3401A (image courtesy of Silicon Labs)
+
+Hardware
+********
+
+- Advanced Energy Monitoring provides real-time information about the energy
+  consumption of an application or prototype design.
+- Ultra low power 128x128 pixel Memory-LCD
+- 2 user buttons, 2 LEDs and two capacitive buttons
+- Humidity, temperature, and inductive-capacitive metal sensor
+- On-board Segger J-Link USB debugger
+
+For more information about the EFM32PG SoC and EFM32PG-STK3401A board:
+
+- `EFM32PG Website`_
+- `EFM32PG1 Datasheet`_
+- `EFM32PG-STK3401A Website`_
+- `EFM32PG-STK3401A User Guide`_
+
+Supported Features
+==================
+
+The efm32pg_stk3402a board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| MPU       | on-chip    | memory protection unit              |
++-----------+------------+-------------------------------------+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | rtcc                                |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c port-polling                    |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+| TRNG      | on-chip    | true random number generator        |
++-----------+------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/efm32pg_stk3401a/efm32pg_stk3401a_defconfig``
+
+Other hardware features are currently not supported by the port.
+
+Connections and IOs
+===================
+
+The EFM32PG1 SoC has twelve GPIO controllers (PORTA to PORTL), but only four
```

Comment:
I think this variant has at most 6 GPIO controllers, PORTA to PORTF.

Reply:
This variant has 5 ports: PORTA-PORTD and PORTF